scripts/verify-bun-review-artifacts.py

```
@@ -7097,6 +7122,16 @@ def require_gate_outcome(root: pathlib.Path) -> None:
         fail(
             f"gate outcome {conclusion} conclusion requires at least one reason"
         )
+    outcome_revision = outcome.get("revision")
+    if outcome_revision is not None:
+        _require_revision_ref(outcome_revision, "gate_outcome.json")
+        graph = load_json(root / "review/claim_graph.json")
+        if isinstance(graph, dict) and graph.get("revision") is not None:
+            if graph["revision"].get("digest") != outcome_revision.get("digest"):
```

Ruff (0.16.2)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)